source/nodejs/adaptivecards-designer/webpack.config.js

```diff
@@ -42,7 +39,7 @@ module.exports = (env, argv) => {
 				exclude: /(node_modules|__tests__)/
 			},
 			{
-				test: /\.css$/,
+				test: /\.css$/i,
```

Comment:
nit: looks like a typo, but i'm certain it's not.

Reply:
this just makes the regex [case-insensitive](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#advanced_searching_with_flags), though it's not really necessary here. I'll back it out